core/src/main/kotlin/com/ritense/valtimo/security/ActuatorSecurityFilterChainFactory.kt

```diff
@@ -73,6 +73,9 @@ class ActuatorSecurityFilterChainFactory {
             }
             .authenticationManager(actuatorAuthenticationManager(passwordEncoder, username, password))
             .httpBasic { it.realmName(ACTUATOR_REALM) }
+            .csrf {
+                csrfConfigurer -> csrfConfigurer.ignoringRequestMatchers("${webEndpointProperties.basePath}/loggers/**")
```

Comment:
Nowadays, we have another endpoint to get the logs:
```
POST /api/management/v1/logging
```

Reply:
This PR is regarding Spring Actuator endpoints, that's exposed for management via e.g. Spring Boot Admin. Also, this endpoint is about setting log levels. See https://docs.spring.io/spring-boot/api/rest/actuator/loggers.html#loggers.setting-level